exonum/src/node/mod.rs

```diff
@@ -793,6 +794,11 @@ impl ApiSender {
         ApiSender(inner)
     }
 
+    /// Creates a dummy sender which cannot send messages.
+    pub fn closed() -> Self {
```

Comment:
> Is this method useful outside of testing purposes?

No, probably not; but the purpose of the method is clearly stated in the doc comment.

> May you mark it as `cfg(test)`?

It is used in `Blockchain::build_for_tests()`, which is not `cfg(test)` since it has uses outside of the crate (e.g., in the `explorer` example / integration test). Do you think this method pollutes the public interface of `ApiSender`?